src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Transition guide for OpenTelemetry cumulative sum metrics
+metaDescription: 'A transition guide about a change in how New Relic handles OpenTelemetry cumulative sum metrics, and how to update your queries and alerts for that change.'
+redirects: 
+---
+
+Our OTLP metric pipeline is being upgraded to better support how we handle [cumulative metrics](/docs/data-apis/understand-data/metric-data/cumulative-metrics). As part of this upgrade we're transitioning the underlying [metric data type](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types) for [monotonic cumulative sums](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#sums) from a `gauge` type to the `cumulativeCount` type. The `cumulativeCount` type allows you to query both the cumulative and delta value associated with reported data. 
+
+Starting March 14, 2023, new accounts will have access to this new `cumulativeCount` type, but if your account was sending cumulative monotonic sums prior to March 14th, you won't have access to that metric type until June 13. On that date, the use of our `gauge` metric type for cumulative sums will be deprecated and reach end-of-life (EoL), and your cumulative sums will be stored as the `cumulativeCount` type. 
```

Comment:
```suggestion
Starting March 14, 2023, we will ingest cumulative monotonic metrics according to the following end-of-life (EoL) timeline:
- March 14th, 2023: The ingest of cumulative monotonic sums as the `cumulativeCount` metric type is released. The prior experience of ingesting these metrics as the `gauge` metric type is deprecated.
  - Accounts that were actively sending cumulative monotonic sums prior to this date will continue to use `gauge` type ingest. 
  - All other accounts will have cumulative monotonic sums ingested as `cumulativeCount` metric types. 
  
- March 14th - June 13th, 2023: Accounts using the deprecated `gauge` type ingest may opt-in to `cumulativeCount` type ingest at any time.

- June 13th, 2023: All accounts will be migrated to `cumulativeCount` type ingest for cumulative monotonic sums.
```
The formatting here is letting me down a bit but does this make sense? The idea was to make the dates/events a bit more readable.